examples/kafka_spark_demo/feature_repo/features.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
+from pyspark.sql import DataFrame
+
+from feast import (
+    FeatureView,
+    Field,
+)
+from feast.stream_feature_view import stream_feature_view
+from feast.types import Float32
+
+from data_sources import *
+from entities import *
+
+driver_hourly_stats_view = FeatureView(
+    name="driver_hourly_stats",
+    description="Hourly features",
+    entities=[driver],
+    ttl=timedelta(seconds=8640000000),
+    schema=[
+        Field(name="conv_rate", dtype=Float32),
+        Field(name="acc_rate", dtype=Float32),
+        Field(name="miles_driven", dtype=Float32),
+    ],
+    online=True,
+    source=driver_stats_batch_source,
+    tags={"production": "True"},
+    owner="test2@gmail.com",
+)
+
+@stream_feature_view(
+    entities=[driver],
+    ttl=timedelta(days=1),
+    mode="spark",
+    schema=[
```

Comment:
does there also need to be the entity in the schema too?

Reply:
not strictly necessary as entity column will be inferred